src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBox/TextBox_ImeOptions.xaml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.TextBoxTests
+{
+	[SampleControlInfo("TextBox")]
+	public sealed partial class TextBox_ImeOptions : UserControl
+    {
+        public TextBox_ImeOptions()
+        {
+            this.InitializeComponent();
+
+#if __ANDROID__
```

Comment:
Could this be a RuntimeTest?

Reply:
Yes, it's better indeed, will change, thanks.